tools/topology/sof-imx8qxp-wm8960.m4

```diff
@@ -60,9 +60,9 @@ dnl PCM_PLAYBACK_ADD(name, pcm_id, playback)
 PCM_PLAYBACK_ADD(Port0, 0, PIPELINE_PCM_1)
 
 dnl DAI_CONFIG(type, idx, link_id, name, sai_config)
-DAI_CONFIG(SAI, 1, 0, SAI1-Codec,
-	SAI_CONFIG(I2S, SAI_CLOCK(mclk, 49152000, codec_mclk_in),
-		SAI_CLOCK(bclk, 3072000, codec_slave),
-		SAI_CLOCK(fsync, 48000, codec_slave),
+DAI_CONFIG(SAI, 1, 0, be.wm8960-hifi,
+	SAI_CONFIG(I2S, SAI_CLOCK(mclk, 12288000, codec_mclk_in),
+		SAI_CLOCK(bclk, 3072000, codec_master),
+		SAI_CLOCK(fsync, 48000, codec_master),
```

Comment:
I presume these are the correct values (does it work?)

Reply:
It does